src/modules/45-dashboards/components/AidaPromptSelection/__tests__/AidaPromptSelection.test.tsx

```diff
@@ -0,0 +1,100 @@
+  const promptContent3 = 'Prompt 3'
+  const promptContent4 = 'Prompt 4'
+
+  const promptTitle1 = 'Prompt Title 1'
+  const promptTitle2 = 'Prompt Title 2'
+
+  test('it should display prompt selection title', async () => {
+    renderComponent({ prompts: [], onPromptSelected: jest.fn(), title })
+
+    expect(screen.getByText(title)).toBeInTheDocument()
+  })
+
+  test('it should display prompt without sub-title', async () => {
+    const prompt: Prompt = {
+      options: [{ content: promptContent1 }]
+    }
+    renderComponent({ prompts: [prompt], onPromptSelected: jest.fn(), title })
+
+    expect(screen.getByText(promptContent1)).toBeInTheDocument()
+    expect(screen.queryByTestId('prompt-title-0')).toBeNull()
+  })
+
+  test('it should display prompt with sub-title', async () => {
+    const prompt: Prompt = {
+      title: promptTitle1,
+      options: [{ content: promptContent1 }]
+    }
+    renderComponent({ prompts: [prompt], onPromptSelected: jest.fn(), title })
+
+    expect(screen.getByText(promptContent1)).toBeInTheDocument()
+    expect(screen.getByTestId('prompt-title-0')).toBeInTheDocument()
+    expect(screen.getByText(promptTitle1)).toBeInTheDocument()
+  })
+
+  test('it should allow prompt to be clicked', async () => {
+    const prompt: Prompt = {
+      title: promptTitle1,
+      options: [{ content: promptContent1 }]
+    }
+    const mockPromptSelect = jest.fn()
+    renderComponent({ prompts: [prompt], onPromptSelected: mockPromptSelect, title })
+
+    expect(screen.getByText(promptContent1)).toBeInTheDocument()
+    const promptButton = screen.getByTestId('prompt-option-0-0')
+
+    act(() => {
+      fireEvent.click(promptButton)
+    })
+    expect(mockPromptSelect).toHaveBeenCalled()
+  })
+
+  test('it should display multiple prompts with multiple options', async () => {
+    const prompt1: Prompt = {
+      title: promptTitle1,
+      options: [{ content: promptContent1 }, { content: promptContent2 }]
+    }
+
+    const prompt2: Prompt = {
+      options: [{ content: promptContent3 }]
+    }
+    const prompt3: Prompt = {
+      title: promptTitle2,
+      options: [{ content: promptContent4 }]
+    }
+    const mockPromptSelect = jest.fn()
+    renderComponent({ prompts: [prompt1, prompt2, prompt3], onPromptSelected: mockPromptSelect, title })
+
+    expect(screen.getByText(promptTitle1)).toBeInTheDocument()
+    expect(screen.getByText(promptContent1)).toBeInTheDocument()
+    expect(screen.getByText(promptContent2)).toBeInTheDocument()
+    expect(screen.getByText(promptContent3)).toBeInTheDocument()
+    expect(screen.getByText(promptTitle2)).toBeInTheDocument()
+    expect(screen.getByText(promptContent4)).toBeInTheDocument()
+
+    const promptButton = screen.getByTestId('prompt-option-2-0')
+    expect(promptButton).toBeInTheDocument()
+    act(() => {
+      fireEvent.click(promptButton)
+    })
+    expect(mockPromptSelect.mock.calls[0][0]).toBe(prompt3.options[0])
```

Comment:
I think you should be able to simplify with `toHaveBeenCalledWith`
```suggestion
    expect(mockPromptSelect).toHaveBeenCalledWith(prompt3.options[0])
```